examples/basic/cmd/calc-cli/http.go

```diff
@@ -10,12 +10,16 @@ import (
 	goahttp "goa.design/goa/http"
 )
 
-func doHTTP(scheme, host string, timeout int, debug bool) (goa.Endpoint, interface{}, error) {
+func doHTTP(scheme, host string, timeout int, acceptType string, debug bool) (goa.Endpoint, interface{}, error) {
```

Comment:
Not sure I understand why this needs to be specified as a parameter, this should all be driven by the design?